src/components/data-management/ProjectsList.jsx

```diff
@@ -0,0 +1,115 @@
+import React, { useState, useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import PropTypes from 'prop-types';
+import { Space } from 'antd';
+
+import ProjectDeleteModal from './ProjectDeleteModal';
+import { setActiveProject, updateProject, deleteProject as deleteProjectAction } from '../../redux/actions/projects';
+import ProjectCard from './ProjectCard';
+import validateInputs, { rules } from '../../utils/validateInputs';
+
+const validationChecks = [
+  rules.MIN_8_CHARS,
+  rules.MIN_2_SEQUENTIAL_CHARS,
+  rules.ALPHANUM_DASH_SPACE,
+  rules.UNIQUE_NAME_CASE_INSENSITIVE,
+];
+
+const ProjectsListContainer = (props) => {
+  const { height, filter } = props;
+  const dispatch = useDispatch();
+
+  const projects = useSelector((state) => state.projects);
+  const [deleteModalVisible, setDeleteModalVisible] = useState(false);
+  const [deleteProjectUuid, setDeleteProjectUuid] = useState(false);
+  const [projectNames, setProjectNames] = useState(new Set());
+
+  const validationParams = {
+    existingNames: projectNames,
+  };
+
+  const searchable = projects.ids.map((projectUuid) => {
+    const project = projects[projectUuid];
+
+    const searchObj = {
+      projectUuid,
+      projectName: project.name,
+      experiments: project.experiments,
+    };
+
+    return [
+      searchObj,
+      project,
+    ];
+  });
```

Comment:
what's the point of this function `searchable`? I am really confused here.

Reply:
It was made to make it easier to search. It's removed in latest refactor.